server/monitor.go

```diff
@@ -980,6 +980,7 @@ func (s *Server) HandleRoot(w http.ResponseWriter, r *http.Request) {
 	<a href=/connz>connz</a><br/>
 	<a href=/routez>routez</a><br/>
 	<a href=/subsz>subsz</a><br/>
+	<a href=/gatewayz>gatewayz</a><br/>
```

Comment:
I would make subsz last and move gateway up one.

Reply:
will do